1264

jackgrossman18 opened the pull request:
Added requested test for conflate button interaction

Commit:
Removed external files from tests

Commit:
Merge branch 'hoot2x-test-fix' into 1264

Commit:
Resolved karma config bug

Commit:
Adding test for conflation button interation

Commit:
Ading async await to delete UndividedHighway layer

putipong reviewed (changes requested):
You got the idea of creating mock data needed for tests down, but you're also repeating tests that have already been implemented in other suites. It's best to avoid this because it makes it more difficult to track down what went wrong if one or the other fails and it causes tests to run longer than needed.

Instead, try manually uploading data by directly invoking the API with params in a `before()` function. Doing this will ensure that you have all the test files you need _before_ the actual tests begin.

Commit:
Disabled broken tests

Commit:
Corrected conflateButton test

Commit:
Updated basemap test to include async deletion of raster file

Commit:
Updated review mode test to create and delete layers within test

Commit:
Update test to create and delete layers within test

Commit:
Removed excess code

Commit:
Removed methods for test layers

Commit:
Test for advOpts panel

Commit:
working on fixing api connection error

Commit:
last test

Commit:
Correcting async upload and deletion methods

Commit:
Updated translations test

Commit:
Updating test

Commit:
Updating tests

Commit:
Updating adv opt panel test

Commit:
Updating reiview bookmark test

Commit:
modified:   ../test/hoot/index.js
	deleted:    ../test/hoot/spec/about.js
	modified:   ../test/hoot/spec/advOpts/advOpts.js
	modified:   ../test/hoot/spec/basemapPanel/addBasemap.js
	modified:   ../test/hoot/spec/conflate/conflateButton.js
	deleted:    ../test/hoot/spec/managePanel.js
	deleted:    ../test/hoot/spec/navbar.js
	modified:   ../test/hoot/spec/reviewMode/reviewMode.js
	modified:   ../test/hoot/spec/sidebar/addLayers/layerColor.js
	modified:   ../test/hoot/spec/translationsPanel/translationAssistant.js

Commit:
modified:   data/imagery.json
	modified:   test/hoot/index.js
	deleted:    test/hoot/spec/about.js
	modified:   test/hoot/spec/advOpts/advOpts.js
	modified:   test/hoot/spec/basemapPanel/addBasemap.js
	modified:   test/hoot/spec/conflate/conflateButton.js
	deleted:    test/hoot/spec/managePanel.js
	deleted:    test/hoot/spec/navbar.js
	modified:   test/hoot/spec/reviewMode/reviewMode.js
	modified:   test/hoot/spec/sidebar/addLayers/layerColor.js
	modified:   test/hoot/spec/translationsPanel/translationAssistant.js

Commit:
Removing unnecessary code
